templates/start-dev-env.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+#
+# This script is to create a dev namespace on Staging environment
+#
+PROJECT_NAME=<% .Name %>
+ENVIRONMENT=stage
+ACCOUNT_ID=<% index .Params `accountId` %>
+REGION=<% index .Params `region` %>
+
+# common functions
+function usage() {
+    echo
+    echo "Usage:"
+    echo "  $0 <IAM user> <project id>"
+    echo "    - IAM user: can be found by running 'aws iam get-group --group-name ${PROJECT_NAME}-developer-${ENVIRONMENT} | jq -r .Users[].UserName'"
+    echo "    - project id: can be 001, 002, or whatever id without space"
+    exit 1
+}
+
+function command_exist() {
+    command -v ${1} >& /dev/null
+}
+
+function error_exit() {
+    echo "ERROR : $1"
+    exit 2
+}
+
+function can_i() {
+    command=$1
+    kubectl auth can-i $1 >& /dev/null || error_exit "No permission to $1"
+}
+
+# Start
+USER_NAME=$1
+DEV_PROJECT_ID=$2
+[[ -z "$USER_NAME" ]] && usage
+[[ -z "$DEV_PROJECT_ID" ]] && DEV_PROJECT_ID="001"
+
+echo '[Dev Environment]'
+
+# Validate cluster
+CLUSTER=$(kubectl config current-context)
+[[ ${CLUSTER} == "${PROJECT_NAME}-${ENVIRONMENT}-${REGION}" ]] || error_exit "Your kubernetes context ${CLUSTER} is not proper to run this script"
```

Comment:
Since we already know the context we need, let's just either change the context here or manually specify the `--context` argument.

Reply:
better use context --context